src/qt/guiutil.cpp

```diff
@@ -777,6 +777,19 @@ QString ConnectionTypeToQString(ConnectionType conn_type)
     assert(false);
 }
 
+QString ConnectionTypeToShortQString(ConnectionType conn_type, bool relay_txes)
+{
+    switch (conn_type) {
+    case ConnectionType::INBOUND: return QObject::tr(relay_txes ? "Full Relay" : "Block Relay");
```

Comment:
```suggestion
    case ConnectionType::INBOUND: return relay_txes ? QObject::tr("Full Relay") : QObject::tr("Block Relay");
```

Reply:
This seems equivalent to me, but thanks for looking! done.

Reply:
It is [not equivalent](https://doc.qt.io/qt-5/i18n-source-translation.html#using-tr-to-obtain-a-translation) for the `lupdate` tool:
> When Qt's translation tool, lupdate, is used to process a set of source files, the text wrapped in `tr()` calls is stored in a section of the translation file that corresponds to its translation context.